Source/Tests/VDSO_Emulation.cpp

```diff
@@ -291,6 +292,21 @@ namespace FEX::VDSO {
     return VDSOBase;
   }
 
+  uint64_t GetVSyscallEntry(void* VDSOBase) {
```

Comment:
seems like this could take a `const void*`

Reply:
Resolved.